src/Data/These.purs

```diff
@@ -119,6 +125,10 @@ maybeThese = case _, _ of
   Just a, Just b -> Just (Both a b)
   Nothing, Nothing -> Nothing
 
+-- | Takes two default values and a `These` value. If the `These` value is
+-- | `This` or `That`, the value wrapped in the `These` value and its
+-- | corresponding default value are wrapped into a `Tuple`.
+-- | Otherwise, the values stored in the `Both` is rewrapped into a `Tuple`.
```

Comment:
```suggestion
-- | corresponding default value are wrapped into a `Tuple`. Otherwise, the
-- | values stored in the `Both` are re-wrapped into a `Tuple`.
```

Minor typo here ("values" -> "are")

Reply:
Fixed